### DISCUSSION TITLE
MFront on ARM-based processors

### DISCUSSION THREAD

**Discussion by mhpierre** [thelfer/tfel#456] [Q&A] [answered]
I tried to use MFront on an ARM-based machine (Apple M1). I was able to install MFront but when compiling an mfront behavior, it seems to be looking to link with x86 libraries. Has MFront been run on ARM by anyone, is there something than I did wrong somewhere in my install ?
When compiling a law I get the following:

`ld: warning: ignoring file /opt/homebrew/Caskroom/miniconda/base/envs/fenicsx-env/lib/libTFELMath.dylib, building for macOS-x86_64 but attempting to link with file built for macOS-arm64`

suggesting that it tries to build for x86_64 even though my machine is arm64, then proceeds to look for symbols for x86 architecture.

`ld: symbol(s) not found for architecture x86_64`

If someone has managed to run on ARM, I would be interested to hear. Otherwise, I will stick with using MFront from a container on my ARM-based machines.

Thanks in advance,
Maxime

**Comment by maxence-wz** [comment, marked as answer]
Hi Maxime, 
did you try to install MFront with Homebrew package manager ? 
`brew install tfel`

**Reply by mhpierre** [reply]
I didn't even think to check if it was on Homebrew! It does work, thanks!
